### DISCUSSION TITLE
Back functionality is not working properly.

### DISCUSSION THREAD

**Discussion by brijesh-imemori** [gilbarbara/react-joyride#1028] [General]
here is the link,
https://jam.dev/c/08e0ffd0-77b1-4264-9d8c-53b64b1a0f33

Issues:
- The issue is whenever we are triggering joyride there is some jerk thing is happening while highlighting on the first widget
- And the second issue is after going to certain widgets the back is not working properly.

please reply us if you got any solution for that.

**Comment by gilbarbara** [comment]
Hey @brijesh-imemori 
This is more of an implementation problem than an issue with the library.
Without a minimal reproducible example is hard to debug anyway.

**Reply by brijesh-imemori** [reply]
Hi @gilbarbara can we discuss on this can you help me fix this?
